app/models/user.rb

```diff
@@ -2,7 +2,7 @@
 
 class User < ApplicationRecord
   has_secure_password
-
   validates :email, presence: true, uniqueness: true
   validates :password, length: { minimum: 6, maximum: 10 }, on: :create
+  validates_presence_of :name, :password_digest, :email
```

Comment:
Good addition of name validation

Could maybe remove password_digest and email? Are they already covered by the other vaildations?